### ISSUE TITLE
render formula failed when using `<embed/>` to embed svg into html

### ISSUE THREAD

**Issue by dzylikecode** [jgraph/drawio#3873]
### Preflight Checklist

* [x] I agree to follow the [Code of Conduct](https://github.com/jgraph/drawio/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for a feature request that matches the one I want to file, without success.

**Describe the bug**

Contrast to other HTML tags like `<iframe>` or `<img/>`, render formula failed when using `<embed/>` to embed SVG into html

| origin canvas | `<iframe>` |`<embed/>`|`<img/>`|
| --- | --- | --- | --- |
|![image](https://github.com/jgraph/drawio/assets/71511961/e98ac202-d0ca-4e42-b2e5-52d7ef4cbb6d)|![image](https://github.com/jgraph/drawio/assets/71511961/e8e57c0c-38fb-4954-8836-ad96e56d9cd3)|![image](https://github.com/jgraph/drawio/assets/71511961/7270077b-20fc-47e2-817c-e8f212247367)|![image](https://github.com/jgraph/drawio/assets/71511961/10cbe2c4-5423-4ac2-81e3-957be3502141)|

```HTML
<!-- use iframe -->
<iframe src="math.svg"></iframe>
```

```HTML
<!-- use embed -->
<embed src="math.svg"/>
```

```HTML
<!-- use img -->
<img src="math.svg" />
```

After drawing 

**To Reproduce**
1. Open the desktop App or browser(no matter which)
2. Draw something you like and type any formula in `Latex` format, for example:
    ```txt
    $$E = mc^2$$
    ```
4. Click `Extras`
5. Enable `Mathematical typesetting`
6. Download the canvas in `SVG` format
8. using `<embed/>` and `<iframe>` to embed SVG into a HTML file respectively
9. see the formula rendered failed when using `<embed/>`

**Expected behavior**

I assume using `<embed/>` will render the formula correctly, just like using `<iframe>`, which is important to me. Because I want to embed SVG into HTML as it's native there without an ugly scrollbar if the SVG is large. If using `<img/>`, the interaction will lost, such as clicking a link.

**Screenshots**

**draw.io version (In the Help->About menu of the draw.io editor):**

- draw.io version 21.7.5

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser chrome
 - Browser Version 116.0.5845.188

**I tested the problem in incognito/private mode with all browser extensions switched off, write "yes" below:**
- yes

**Additional context**

I have tried to use Katex to replace MathJax when [developing an extension for supporting markdown syntax in drawio](https://github.com/dzylikecode/drawio-plugin-markdown). It can render the formula correctly if using the `<embed/>`. 
Here is the effect of using Katex to replace MathJax and using `<embed/>` to embed SVG into HTML. The formula is rendered well.

![image](https://github.com/jgraph/drawio/assets/71511961/69c05ff3-b48f-4322-9b7b-cfb265f2c208)

But I can't import the whole CSS of Katex into drawio, so some formula is so ugly after rendering. I have to give up the idea and ask for your help to solve the problem. Thank you!
| ugly |
| --- |
|![image](https://github.com/jgraph/drawio/assets/71511961/2f488a11-20c1-4295-ad4d-eb92d0872e10)|

- the same error occurs when using the API `MathJax.tex2svg`, so there is something wrong with SVG format
- nothing to show if using the API `MathJax.tex2mml`

**Comment by davidjgraph** [comment]
Is there a public example we can look at?

**Comment by dzylikecode** [comment]
The bug has been solved after updating to v23.1.5.
